scripts/pylib/twister/twisterlib/environment.py

```diff
@@ -597,6 +597,17 @@ def add_parse_arguments(parser = None):
         help="Use the list of test scenarios under quarantine and run them"
              "to verify their current status.")
 
+    parser.add_argument(
+        "--scripting-list",
+        action="append",
+        metavar="FILENAME",
+        help="The feature is exclusively compatible with hardware-based "
```

Comment:
`is exclusively compatible` - this needs to be implemented as a check for required options - see `parse_arguments()`

Reply:
added ` if options.scripting_list and not options.device_testing:`